tools/ci_build/github/azure-pipelines/templates/win-ci-2019.yml

```diff
@@ -195,9 +196,10 @@ jobs:
       # Esrp signing
       - template: win-esrp-dll.yml
         parameters:
-          FolderPath: '$(Build.BinariesDirectory)\$(BuildConfig)'
+          FolderPath: '$(Build.BinariesDirectory)\$(BuildConfig)\$(BuildConfig)'
```

Comment:
Is `$(BuildConfig)\$(BuildConfig)'` intentional?

Reply:
Yes. Otherwise too many files could be submitted. 

$(Build.BinariesDirectory)\$(BuildConfig) is where onnxruntime.sln sits
$(Build.BinariesDirectory)\$(BuildConfig)\$(BuildConfig) is the parent folder of our binaries.